src/scripts/services/validation-service.ts

```diff
@@ -20,5 +22,32 @@ export const validateGroupExists = (
 ): E.Either<NEA.NonEmptyArray<string>, Group> =>
   pipe(
     maybeGroup,
-    E.fromOption(() => NEA.of(`Group [${group}] does not exist`))
+    E.fromOption(() => NEA.of(`Group [${group}] does not exist.`))
+  );
+
+/**
```

Comment:
**issue:** this doesn't seem like a good fit. The rest of this service is just manipulating values - no external calls. Move this to `okta-service.ts` or similar please.

Reply:
modified in commit: 86b86bb42c3c933247cfd80ab13535eb1eed5de5